docs/en/setup/service-agent/java-agent/configurations.md

```diff
@@ -103,6 +103,8 @@ This is the properties list supported in `agent/config/agent.config`.
 | `plugin.kafka.namespace`                                        | isolate multi OAP server when using same Kafka cluster (final topic name will append namespace before Kafka topics with `-` ).                                                                                                                                                                                                                                                                                                                                                                                                                         | SW_KAFKA_NAMESPACE                                               | ``                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+| `plugin.kafka.decrypt_class`                                    | Specify which class to decrypt encrypted configuration of kafka.You can set encrypted information in `plugin.kafka.producer_config_json` or `plugin.kafka.producer_config` if you need.                                                                                                                                                                                                                                                                                                                                                                | SW_KAFKA_DECRYPT_CLASS                                           | ``                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
```

Comment:
https://skywalking.apache.org/docs/skywalking-java/next/en/setup/service-agent/java-agent/advanced-reporters/

This doc should be updated as well, including
1. Which artifact and version should the developer depend on to develop this? Which interface should be implemented?
2. Where to put the packaged codes